### ISSUE TITLE
spelling issue: artifically  ==> artificially

### ISSUE THREAD

**Issue by pabs3** [mvp/uhubctl#126]
```
$ codespell --quiet-level=3 .
./uhubctl.c:63: artifically  ==> artificially
```

**Comment by pabs3** [comment]
Also, a capitalisation issue in the README:

```
$ spellintian ./README.md
./README.md: OpenSUSE -> openSUSE
```

**Comment by mvp** [comment]
Fixed in cfee02f014ae02d343295b412161b3bd095da01f.
